src/streams/send.rs

```diff
@@ -680,6 +681,28 @@ impl StreamTx {
         //    first ever poll_packet().
         let use_for_padding = self.is_audio.map(|i| !i).unwrap_or(false);
 
+        let mut snapshot = self.send_queue.snapshot(now);
+
+        // Fake some stats for padding.
+        if self.padding > 0 {
+            // TODO: Be more scientific about this factor.
+            const AVERAGE_PADDING_PACKET_SIZE: usize = 200;
+
+            snapshot.priority = if snapshot.packet_count == 0 {
+                QueuePriority::Padding
+            } else {
+                QueuePriority::Media
+            };
+
+            let fake_packets = self.padding / AVERAGE_PADDING_PACKET_SIZE;
+
+            snapshot.size += self.padding;
+            snapshot.packet_count += fake_packets as u32;
+
+            // TODO: Should this be faked in relation to size of padding?
```

Comment:
My removal of the VecDeque we used for `self.padding` probably doesn't help here. If we still had that we could keep a `queued_at: Instant` field with every padding request and produce a better value here.

To answer the TODO: Shouldn't it be scaled with the number of packets?

Reply:
Yeah. I was thinking that too. However. Maybe it's an all-or-nothing situation. In the sense that either we make perfect stats (like we did in the past), or we get away with not perfect stats. I do really like the simplification you did.

Reply:
Yea, this is still an improvement. If we just scale it with the number of packets maybe that'd be okay for now